source/includes/_appendix.md

```diff
@@ -329,600 +329,23 @@ zoominfo|<span class="yes">Yes</span>||<span class="yes">Yes</span>|<span class=
 
 
 
-## Business Category IDs
```

Comment:
Rather than completely removing this section maybe leave the heading and link to the relevant endpoint elsewhere in the docs. This will allow people who are used to referring to this section to find the new endpoint more easily.